README.md

```diff
@@ -19,6 +22,16 @@ There are two additional functionalities added which supports binary and label i
 
 - **MicrojsonBinaryModel:** Reconstruct binary images using polygon coordinates from json file.
 
+## Installation
+
+To install MicroJSON, you can use the following command:
+
+```pip install microjson```
+This will install the default version of MicroJSON with the basic functionalities and minimal dependencies. If you want to use the additional functionalities, such as provided by the ```utils``` module, you can install the package with the following command:
+
+```pip install microjson[all]```
+
+
 ## Usage
 
 MicroJSON can be used with any application or tool that can process JSON data. Due to its design, it is particularly suited to applications related to the analysis, visualization, and manipulation of microscopy images.
```

Comment:
This can be rewritten if you want
MicroJSON is compatible with any application or tool that process JSON data. Its design makes it especially well-suited for applications involving analysis, visualization, and manipulation of microscopy images